### PR TITLE
docs: add llm kmeans notebook as an included example

### PR DESCRIPTION
**Pull request by milkshakeiii**
Original notebook written by Jiaxunwu.

I have tested the notebook by running it on Colab with a test prod project. Any feedback on content or format would be appreciated. I didn't change very much code from Jiaxun's original. Thanks!

### CODE REVIEW

**Review by ashleyxuu — COMMENTED** [review]
Would you mind adding a "Clean up" section like: https://github.com/googleapis/python-bigquery-dataframes/blob/main/notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb.

nit: We should also be consistent with this structure, adding: Installation, authentication, etc... section

File: notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+      "source": [
+        "### Costs\n",
+        "\n",
+        "This tutorial uses billable components of Google Cloud:\n",
+        "\n",
+        "* BigQuery (compute)\n",
+        "* BigQuery ML\n",
+        "\n",
+        "Learn about [BigQuery compute pricing](https://cloud.google.com/bigquery/pricing#analysis_pricing_models),\n",
+        "and [BigQuery ML pricing](https://cloud.google.com/bigquery/pricing#bqml),\n",
```
Comment (ashleyxuu) [inline notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb:92]: For cost, we should also add add Vertex/PaLM https://screenshot.googleplex.com/5qQxkgsVCymCgkH

File: notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+        "2. Use KMeans clustering to group together complaints whose text embeddings are near to eachother. This will give us sets of similar complaints, but we don't yet know _why_ these complaints are similar.\n",
+        "3. Simply ask PaLM2TextGenerator in English what the difference is between the groups of complaints that we got. Thanks to the power of modern LLMs, the response might give us a very good idea of what these complaints are all about, but remember to [\"understand the limits of your dataset and model.\"](https://ai.google/responsibility/responsible-ai-practices/#:~:text=Understand%20the%20limitations%20of%20your%20dataset%20and%20model)\n",
```
Comment (ashleyxuu) [inline notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb:64]: nit: I would change "Simply ask" to "Prompt the PaLM2TextGenerator in English asking what.."

File: notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb (hunk 1 above)
Comment (milkshakeiii) [reply notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb:92]: Added, thanks.

File: notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb (hunk 2 above)
Comment (milkshakeiii) [reply notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb:64]: True, PaLM2TextGenerator can't exactly be asked questions like a person, ha. Fixed.

**Comment by milkshakeiii** [comment]
> Would you mind adding a "Clean up" section like: https://github.com/googleapis/python-bigquery-dataframes/blob/main/notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb.
> 
> nit: We should also be consistent with this structure, adding: Installation, authentication, etc... section

Many of those sections are not needed anymore or not needed for this notebook right? For example `!pip install -U --quiet bigframes` is not needed any more I think? I will work on adding the clean up section and any other ones that are still needed.

**Comment by milkshakeiii** [comment]
> > Would you mind adding a "Clean up" section like: https://github.com/googleapis/python-bigquery-dataframes/blob/main/notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb.
> > nit: We should also be consistent with this structure, adding: Installation, authentication, etc... section
> 
> Some of those sections are not needed anymore or not needed for this notebook right? For example `!pip install -U --quiet bigframes` is not needed any more I think? I will work on adding the clean up section and any other ones that are still needed.

Ok! I have added all the sections that I think are needed, including all the code needed for the Vertex AI connection. The clean up section luckily it sounds like is not necessary based on discussion with @GarrettWu . I have tested the most recent version of the notebook in Colab. Based on discussion with @shobsi , we have deny listed this type of notebook from being run by the e2e tester, so I deny listed this one as well. Looks like we'll have to rely on manual testing for the time being.

Ready for re-review. Thank you!

**Review by jiaxunwu — APPROVED** [review]

**Review by ashleyxuu — APPROVED** [review]

**Pull request merged**